src/Command/ApiCompare.php

```diff
@@ -135,6 +137,13 @@ public function execute(InputInterface $input, OutputInterface $output) : int
             );
 
             (new SymfonyConsoleTextFormatter($output))->write($changes);
+
+            $markdownFile = trim((string)$input->getOption('markdown'));
+            if ($markdownFile !== '') {
+                $output->write(sprintf('Generating markdown in %s...', $markdownFile));
```

Comment:
We should write to `STDOUT` and `STDERR` separately: https://gist.github.com/MattKetmo/3665488